Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -288,16 +288,19 @@ struct TranslateResult
 };
 TranslateResult JitCache_TranslateAddress(u32 address);
 
-static const int BAT_INDEX_SHIFT = 17;
+constexpr int BAT_INDEX_SHIFT = 17;
```

Comment:
I've added one. Do you also want me to use it for all kind of: `x * size` instead of `x << shift`?